psscriptanalyzer/Common-Helpers.psm1

```
@@ -0,0 +1,58 @@
+$Global:HelperCache = @{}
+
+function Get-Configs {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-Configs' does not have a help comment.

(PSProvideCommentHelp)